src/LocalAppSettings.ts

```diff
@@ -80,12 +80,13 @@ export async function getLocalAppSettings(localSettingsPath: string, allowOverwr
                 return <ILocalAppSettings>JSON.parse(data);
             } catch (error) {
                 if (allowOverwrite) {
-                    const message: string = localize('failedToParse', 'Failed to parse local settings: {0}. Overwrite?', parseError(error).message);
+                    const message: string = localize('failedToParseWithOverwrite', 'Failed to parse "{0}": {1}. Overwrite?', localSettingsFileName, parseError(error).message);
```

Comment:
Pretty confident that a parse failure is the only error to show up here?

Reply:
The only code I'm calling within the `try` block is `JSON.parse` - so yes